unify_emphasis.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# regex = [Ss]ee the {[A-Za-z]*DocURL}\[[_]*Managing Hosts[_]*\] [Gg]uide
+
+# list of docURLs:
+  # AdministeringDocURL
+  # ConfiguringAnsibleDocURL
+  # ConfiguringLoadBalancerDocURL
+  # ContentManagementDocURL
+  # InstallingProjectDocURL
+  # InstallingSmartProxyDocURL
+  # ManagingConfigurationsPuppetDocURL
+  # ManagingHostsDocURL
+  # ManagingOrganizationsLocationsDocURL
+  # PlanningDocURL
+  # ProvisioningDocURL
+  # UpgradingDocURL
+
+# list of DocTitles:
+  # AdministeringAnsibleDocTitle
+  # AdministeringDocTitle
+  # AppCentricDeploymentDocTitle
+  # ConfiguringAnsibleDocTitle
+  # ConfiguringLoadBalancerDocTitle
+  # ContentManagementDocTitle
+  # ConvertingHostRHELDocTitle
+  # DeployingAWSDocTitle
+  # InstallingServerDisconnectedDocTitle
+  # InstallingServerDocTitle
+  # InstallingSmartProxyDocTitle
+  # ManagingConfigurationsPuppetDocTitle
+  # ManagingHostsDocTitle
+  # ManagingOrganizationsLocationsDocTitle
+  # PlanningDocTitle
+  # ProvisioningDocTitle
+  # QuickstartDocTitle
+  # ReleaseNotesDocTitle
+  # UpgradingDocTitle
+
+
+git restore guides/
+
+echo "with the; with guide"
+find guides/ -type f -iname "*\.adoc" | xargs sed -i "" "s/[Ss]ee the {[A-Za-z]*DocURL}\[[_]*Managing Hosts[_]*\] [Gg]uide/see {ManagingHostsDocURL}[_{ManagingHostsDocTitle}_]/g"
```

Comment:
A tiny suggestion - use a backreference for the initial letter to keep caps (just in case they're used somewhere):
`sed -i "" "s/\([Ss]\)ee the {[A-Za-z]*DocURL}\[[_]*Managing Hosts[_]*\] [Gg]uide/\1ee {ManagingHostsDocURL}[_{ManagingHostsDocTitle}_]/g"`

Reply:
FYI, the script isn't working in bash, the first argument `""` after `-i` is excessive (when I removed it, it worked).

Reply:
I ran this script on a non-GNU version of sed; so I need the `""`. In the end, I'd rather add some `find | xargs sed` commands to my commit message than checking in this script.

Yes, good idea to use backreference/ capture groups!

Reply:
Cool as long as the script works. Thank you!